workflows/scripts/normalize.py

```diff
@@ -40,36 +40,27 @@ def normalization(
 
     # loading parameters
     logging.info(f"Loading Annotation configuration from: {config}")
+    normalize_config = config["params"]
 
-    normalize_obj = Path(config)
-    normalize_config_path = normalize_obj.absolute()
-    if not normalize_obj.is_file():
-        e_msg = "Unable to find Normalization configuration file"
-        logging.error(e_msg)
-        raise FileNotFoundError(e_msg)
-
-    with open(normalize_config_path, "r") as yaml_contents:
-        normalize_config = yaml.safe_load(yaml_contents)["normalize_configs"]["params"]
-        logging.info("Annotation configuration loaded")
-
-    meta_features = [
-        "Metadata_Plate",
-        "Metadata_Well",
-        "Metadata_WellRow",
-        "Metadata_WellCol",
-        "Metadata_gene_name",
-        "Metadata_pert_name",
-        "Metadata_broad_sample",
-        "Metadata_cell_line",
-    ]
+    # -- this is form old version of CellProfiler
+    # meta_features = [
```

Comment:
Just wondering, where did you get these metadata column names from? Based on my experience, the metadata columns will always vary by project so I assume that somewhere if you need to access metadata that this is made more flexible?

Reply:
This was obtained from the [cell-health study](https://github.com/broadinstitute/cell-health/blob/30ea5de393eb9cfc10b575582aa9f0f857b44c59/1.generate-profiles/generate_profiles.py#L62-L71) that used hard coded feature names. I think this will no longer be needed anymore since we can write this down in the config file.